src/components/MainLayout.js

```diff
@@ -101,5 +101,53 @@ function Logo() {
 
 function Hamburger({ onClick, active }) {
   // @todo #1 Implement Hamburger menu according to design.
-  return <button onClick={onClick}>{active ? "🍟" : "🍔"}</button>
+
+  const styles = {
+    container: {
+      height: "20px",
+      width: "20px",
+      flexDirection: "column",
+      justifyContent: "center",
+      alignItems: "center",
+      cursor: "pointer",
+      paddingTop: "6px",
+    },
+    line: {
+      height: "2px",
+      width: "20px",
+      background: "#212121",
+      transition: "all 0.2s ease",
+    },
+    lineTop: {
+      transform: active ? "rotate(45deg)" : "none",
+      transformOrigin: "top left",
+      marginBottom: "5px",
+    },
+    lineMiddle: {
+      opacity: active ? 0 : 1,
+      transform: active ? "translateX(-16px)" : "none",
+    },
+    lineBottom: {
+      transform: active ? "translateX(-1px) rotate(-45deg)" : "none",
+      transformOrigin: "top left",
+      marginTop: "5px",
+    },
+  }
+  return (
+    <button onClick={onClick}>
+      {active ? (
```

Comment:
The code for both `true` and `false` cases are the same.

You can replace this conditional with a simple div krub.